-------------------------------------------------------------------------------
Test set: avantica.app.s3.utils.TransactionRepositoryTest
-------------------------------------------------------------------------------
Tests run: 2, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 8.731 s <<< FAILURE! - in avantica.app.s3.utils.TransactionRepositoryTest
findByName(avantica.app.s3.utils.TransactionRepositoryTest)  Time elapsed: 0.332 s  <<< ERROR!
org.springframework.dao.IncorrectResultSizeDataAccessException: query did not return a unique result: 2; nested exception is javax.persistence.NonUniqueResultException: query did not return a unique result: 2
	at avantica.app.s3.utils.TransactionRepositoryTest.findByName(TransactionRepositoryTest.java:51)
Caused by: javax.persistence.NonUniqueResultException: query did not return a unique result: 2
	at avantica.app.s3.utils.TransactionRepositoryTest.findByName(TransactionRepositoryTest.java:51)

